src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/fee/FeeDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+
+public class FeeDto {
+
+    @JsonProperty("calculated_amount")
```

Comment:
Instead of doing this you can set a strategy for the whole class as we do elsewhere:
`@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)`